src/commonTest/kotlin/operations/numeric/compare/GreaterThanTest.kt

```diff
@@ -1,11 +1,12 @@
-package operations.numeric
+package operations.numeric.compare
 
 import JsonLogicEngine
 import TestInput
 import io.kotest.core.spec.style.FunSpec
 import io.kotest.datatest.withData
 import io.kotest.matchers.shouldBe
 
+// testy na opakowane pojedyncze wartosci
```

Comment:
eng pls (if this comment is in fact needed)

Reply:
definitely not needed :D